protocol/x/listing/keeper/grpc_query_pml_enabled_status.go

```diff
@@ -0,0 +1,18 @@
+package keeper
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/dydxprotocol/v4-chain/protocol/x/listing/types"
+)
+
+func (k Keeper) PermissionlessMarketListingStatus(
```

Comment:
```suggestion
func (k Keeper) GetPermissionlessMarketListingStatus(
```

Reply:
Have a look at the other GRPC query keeper methods. We don't add a "Get" to any of those